ui/lib/pki/addon/controllers/roles/index.js

```diff
@@ -0,0 +1,8 @@
+import Controller from '@ember/controller';
+import { getOwner } from '@ember/application';
+
+export default class BlogPostController extends Controller {
+  get mountPoint() {
+    return getOwner(this).mountPoint;
```

Comment:
we need the mountPoint because this is an engine and for the LinkedBlock component to work it needs to know where in the App it is. This is passed through as a param called linkPrefix.